velox/exec/tests/RowContainerTest.cpp

```diff
@@ -32,44 +32,149 @@ using namespace facebook::velox::test;
 
 class RowContainerTest : public exec::test::RowContainerTestBase {
  protected:
-  void testExtractColumnForOddRows(
+  void testExtractColumn(
       RowContainer& container,
       const std::vector<char*>& rows,
       int column,
-      const VectorPtr& expected) {
+      const VectorPtr& expected,
+      int offset = 0) {
```

Comment:
Do we need this parameter? Can we just hard-code it to something, like 10, inside the function?

Reply:
I've changed this code to just set offset as size/3 so that we skip the first 1/3rd of rows.